crates/red_knot_python_semantic/src/types.rs

```diff
@@ -296,6 +297,42 @@ impl<'db> Type<'db> {
         }
     }
 
+    /// Return true if this type is assignable to type `other`.
+    #[allow(unused)]
+    pub(crate) fn is_assignable_to(self, db: &'db dyn Db, other: Type<'db>) -> bool {
```

Comment:
Is there a name that makes it more specific name than `other` that helps clarify the order of assignment (self is assigned to `other`?. Maybe `target`?

```suggestion
    pub(crate) fn is_assignable_to(self, db: &'db dyn Db, target: Type<'db>) -> bool {
```

Reply:
I thought the method name `is_assignable_to` already made the directionality quite clear? But I don't mind renaming the argument to `target`.